File: java/Driver.java
import java.text.MessageFormat;

public class Driver extends Account {
    public int yearsWorking;

    public Driver (String name, String document, int yearsWorking) {
        super(name, document);
        this.yearsWorking = yearsWorking;
    }

    public void greeting() {
        System.out.println(MessageFormat.format("Soy el conductor {0} y llevo {1} años trabajando", this.name, this.yearsWorking));
    }
}

File: python/driver.py
from account import Account

class Driver (Account):
    yearsWorking = int

    def __init__(self, name, document, yearsWorking):
        super().__init__(name, document)
        self.yearsWorking = yearsWorking

    def greeting(self):
        print("Soy el conductor {0} y llevo {1} años trabajando".format(self.name, self.yearsWorking))